kube-state-metrics/main.libsonnet

```
@@ -474,6 +463,38 @@ local d = import 'github.com/jsonnet-libs/docsonnet/doc-util/main.libsonnet';
     deployment+:
       deployment.configMapVolumeMount(self.CRSMConfigMap, '/crsmconfig'),
   },
-
+  withProbes(enabled=true):: {
+    probes_enabled:: true,
+  },
```

I'd call this directly, the boolean adds an unnecessary level of indirection, I'd avoid that so folks don't have to jump around in the code to understand what it does.
```suggestion
  withProbes():: 
      root.withLivenessProbe(port=ksmPortName)
      + root.withReadinessProbe(port=telemetryPortName,
```